openmdao/examples/test/test_examples.py

```diff
@@ -4,8 +4,10 @@
    change to the example file.
 """
 
+import inspect
 import sys
 import math
+import pep8
```

Comment:
Sorry I'll ditch the dependency for now.  Let's use it in the next version but avoid it in clippy.